scipy/stats/stats.py

```diff
@@ -1793,11 +1800,9 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import pylab as pl
```

Comment:
This should be ``import matplotlib.pyplot as plt``. That's what all other examples do. Pylab is a relic and should be avoided for any new code.